ecllibrary/teststd/uni/TestExcludeNthWord.ecl

```diff
@@ -0,0 +1,65 @@
+﻿/*##############################################################################
+ 
```

Comment:
whitespace: this line and the other blank lines in the comment have trailing spaces.  (git diff HEAD^ will highlight them)